### PR TITLE
vapoursyntheditor: Add version 19

### PR DESCRIPTION
**Pull request by SpecterShell**

### CODE REVIEW

**Comment by pratikpc** [comment]
@SpecterShell there is a GitHub repo with releases as well
https://github.com/vapoursynth/vapoursynth

**Comment by pratikpc** [comment]
@SpecterShell the version the GitHub repo has is higher as well

**Comment by SpecterShell** [comment]
> @SpecterShell there is a GitHub repo with releases as well [vapoursynth/vapoursynth](https://github.com/vapoursynth/vapoursynth)

This pr is VapourSynth**Editor**, not VapourSynth.

File: bucket/vapoursyntheditor.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "version": "r19",
```
Comment (rashil2000) [inline bucket/vapoursyntheditor.json:2]: The 'r' is not required, only 19 is sufficient. You will need to change the URLs and checkver too for this.

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Review by rashil2000 — APPROVED** [review]

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**